doc/man3/EVP_md2.pod

```diff
@@ -16,6 +16,10 @@ EVP_md2
 MD2 is a cryptographic hash function standardized in RFC 1319 and designed by
 Ronald Rivest.
 
+=head2 Identity
```

Comment:
I'm not really sure I understand what "Identity" means in this context. I would probably drop this, and just make the sentence below part of the description (and for the other similar occurrence below)

Reply:
I followed the flow in "doc/man7/EVP_MD-MD2.pod" so it  looks the same.
You want me drop it?

Reply:
Ah, but there the full sentence is:

    This implementation is only available with the legacy provider, and is identified with the name "MD2".

So, there the "identity" is the name "MD2" and so the title makes sense. Here you have dropped the bit from the sentence which is the identity (because it is not relevant in this context) and so the title no longer makes sense. Please just drop the title.